EIPS/eip-1.md

```diff
@@ -135,113 +134,87 @@ Random J. User
 
 if the email address is not given.
 
-Note: The resolution header is required for Standards Track EIPs only. It contains a URL that should point to an email message or other web resource where the pronouncement about the EIP is made.
+Note: The resolution header is required for Standards Track EEPs only. It contains a URL that should point to an email message or other web resource where the pronouncement about the EEP is made.
 
-While an EIP is a draft, a discussions-to header will indicate the mailing list or URL where the EIP is being discussed. As mentioned above, examples for places to discuss your EIP include [Ethereum topics on Gitter](https://gitter.im/ethereum/topics), an issue in this repo or in a fork of this repo, [Ethereum Magicians](https://ethereum-magicians.org/) (this is suitable for EIPs that may be contentious or have a strong governance aspect), and [Reddit r/ethereum](https://www.reddit.com/r/ethereum/). No discussions-to header is necessary if the EIP is being discussed privately with the author.
+While an EEP is a draft, a discussions-to header will indicate the mailing list or URL where the EEP is being discussed. As mentioned above, examples for places to discuss your EEP include [FLAG AS THERE IS CURRENTLY NOT A GITTER SETUP?] [EOS topics on Gitter](https://gitter.im/eos/topics), an issue in this repo or in a fork of this repo, [FLAG, IS THERE AN EQUIVALENT TO SET UP?][Ethereum Magicians](https://ethereum-magicians.org/) (this is suitable for EEPs that may be contentious or have a strong governance aspect), and [Reddit r/eos](https://www.reddit.com/r/eos/). No discussions-to header is necessary if the EEP is being discussed privately with the author. [FLAG - WE DON'T HAVE EQUIVALENTS OF GITTER/MAGICIAN SITE]
```

Comment:
> [FLAG AS THERE IS CURRENTLY NOT A GITTER SETUP?]

Telegram I would say.

> [FLAG, IS THERE AN EQUIVALENT TO SET UP?]

Hard to say for me, but IMO, should be turned in a way that it involves the current BPs